code/game/machinery/computer/abnormality_queue.dm

```diff
@@ -81,7 +81,7 @@
 			UpdateAnomaly(target_type, "fucked it lets rolled", TRUE)
 		if("hardcore_fuck_it_lets_roll")
 			if (SSabnormality_queue.hardcore_roll_enabled)
-				var/mob/living/simple_animal/hostile/abnormality/target_type = pick(pick(SSabnormality_queue.possible_abnormalities))
+				var/mob/living/simple_animal/hostile/abnormality/target_type = pick(SSabnormality_queue.possible_abnormalities)
```

Comment:
SSabnormality_queue.possible_abnormalities is a list of lists, it is supposed to double pick. Otherwise you end up with the value being... "TETH"

Reply:
ah, gotcha. Reverting it